picard/script/functions.py

```diff
@@ -1175,7 +1176,7 @@ def func_foreach(parser, multi, loop_code, separator=MULTI_VALUED_JOINER):
 Standard 'while' loop. Executes `code` repeatedly until `condition` no longer
     evaluates to `True`. For each loop, the count is stored in the tag
     `_loop_count`. This allows the count value to be accessed within the `code`
-    script. The function limites the maximum number of iterations to 1000 as a
+    script. The function limits the maximum number of iterations to 1000 as a
```

Comment:
Even though that's a trivial change it shouldn't be in this PR. I did in d13d89b3433c7791c5e27e50c6532912259d8fa5 (pushed to master branch).

Reply:
Sorry about that.